src/Symfony/Bundle/SecurityBundle/DependencyInjection/SecurityExtension.php

```diff
@@ -246,8 +247,9 @@ private function createFirewall(ContainerBuilder $container, $id, $firewall, &$a
         } elseif (isset($firewall['pattern']) || isset($firewall['host'])) {
             $pattern = isset($firewall['pattern']) ? $firewall['pattern'] : null;
             $host = isset($firewall['host']) ? $firewall['host'] : null;
+            $port = isset($firewall['port']) ? $firewall['port'] : null;
             $methods = isset($firewall['methods']) ? $firewall['methods'] : array();
-            $matcher = $this->createRequestMatcher($container, $pattern, $host, $methods);
+            $matcher = $this->createRequestMatcher($container, $pattern, $host, $port, $methods);
```

Comment:
`$firewall['port'] ?? null`, removing the assignment above

Reply:
Can I do the same things for the rest or is it must be another P.R.?

Reply:
I would keep them unchanged to avoid conflicts when merging lower branches up to master